src/roktManager.ts

```diff
@@ -111,7 +121,23 @@ export default class RoktManager {
         this.processMessageQueue();
     }
 
-    public selectPlacements(options: IRoktSelectPlacementsOptions): Promise<IRoktSelection> {
+    /**
+     * Selects placements based on the provided options.
+     * This method handles user identity updates and attribute mapping before selecting placements.
```

Comment:
I think this comment sounds vague but also misleading. Yes it does handle user identity updates and attribute mappings, but only as a side effect.